Python/Object_detection/readme.md

```diff
@@ -0,0 +1,27 @@
+# Object Detection
+  A really basic code for Object Detection using OpenCV. 
+  
+## Dependency
+
+* OpenCV
+* Numpy
+* Jupyter and associated packages.
+* Download the COCO dataset. 
+
+## Setup instructions
+
+1. A virtual environment (recommended)
+2. pip install OpenCV and Numpy
```

Comment:
Put opencv and numpy in `requirements.txt`.

Use this [blog](https://blog.usejournal.com/why-and-how-to-make-a-requirements-txt-f329c685181e).